### PR TITLE
fix: Update XCTestCase proxy to support Xcode 11.4

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
https://github.com/appium/appium/issues/13949

### CODE REVIEW

File: WebDriverAgentLib/Utilities/FBFailureProofTestCase.m [hunk 1]
```diff
@@ -23,7 +23,15 @@ - (void)setUp
 {
   [super setUp];
   self.continueAfterFailure = YES;
-  self.internalImplementation = (_XCTestCaseImplementation *)[FBXCTestCaseImplementationFailureHoldingProxy proxyWithXCTestCaseImplementation:self.internalImplementation];
+  if ([self respondsToSelector:@selector(internalImplementation)]) {
```
Comment (KazuCocoa) [inline WebDriverAgentLib/Utilities/FBFailureProofTestCase.m:26]: How about leave a comment Xcode 11.4 no longer have this API in L17 in PrivateHeaders/XCTest/XCTestCase.h ?

**Review by KazuCocoa — APPROVED** [review]
I tested with Xcode 11.4, iOS 13.4 and iOS 11.4 simulators and `/source` endpoint. This change worked. Seems good to me.

File: WebDriverAgentLib/Utilities/FBFailureProofTestCase.m (hunk 1 above)
Comment (mykola-mokhnach) [reply WebDriverAgentLib/Utilities/FBFailureProofTestCase.m:26]: added

Comment (chenneyhuang) [reply WebDriverAgentLib/Utilities/FBFailureProofTestCase.m:26]: I updated the codes but the error still exists..... I'm using Xcode 11.4, IOS 11.1. In the WDA file I downloaded, the file FBFailureProofTestCaseTest.m was under path WebDriverAgentTests/IntegrationTests, not WebDriverAgentLib/Utilities. Did I download the wrong file?

Comment (KazuCocoa) [reply WebDriverAgentLib/Utilities/FBFailureProofTestCase.m:26]: This file was `WebDriverAgentLib/Utilities/FBFailureProofTestCase.m`, not `FBFailureProofTestCaseTest.m`.
Did you find another issue in `FBFailureProofTestCaseTest.m`?

Comment (chenneyhuang) [reply WebDriverAgentLib/Utilities/FBFailureProofTestCase.m:26]: No. It works fine now. Sorry. That was my stupid mistake. Thanks.

**Pull request merged**